src/main/java/org/trigon/hardware/RobotHardwareStats.java

```diff
@@ -1,37 +1,69 @@
 package org.trigon.hardware;
 
+/**
+ * A class that contains stats about the robot's hardware.
+ */
 public class RobotHardwareStats {
     private static boolean IS_SIMULATION = false;
     private static boolean IS_REPLAY = false;
     private static double PERIODIC_TIME_SECONDS = 0.02;
 
+    /**
+     * Sets the current robot stats.
+     *
+     * @param isReal     whether the robot is real or a simulation
```

Comment:
explain where this should be taken from
explain why use this structure, and which steps are taken to ensure the hardware stats are true